app/models/destination.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Destination < ApplicationRecord
```

Comment:
Estava pensando aqui será que precisamos mesmo desse modelo / tabela, pois se analisarmos bem a `flight_details` já possui a informação do destino e toda vez que é realizado uma busca já é feito a criação desse objeto, imagino que poderiamos usar diretamente o que já temos. O que você acha?

Reply:
:thinking: O FlightDetail é salvo no banco apenas quando ele não existe, de toda forma teremos que adicionar alguma tabela para registar quantas vezes esse destino é pesquisado e a data que ele foi pesquisado, por isso achei melhor criar um modelo exclusivo para isso.
Acha melhor mudar essa estrategia e alterar os modelos já existentes (`Flight` ou `FlightDetail`) para incluir essas informações de quando determinado destino for buscado?

Reply:
Uhm, verdade ele só será salvo na primeira busca...pode ignorar esse comentário.